docs/team/lfrostbytee.md

```diff
@@ -28,20 +28,34 @@ Given below are my contributions to the project.
     * Added the `findEvents` command [\#128](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/128)
         * What it does: Displays all events that match the keywords that have been inputted.
         * Justification: This feature helps the user to better locate the events that they are looking for if they are already managing many events.
+    * Implemented automated testing for `DateTest`, `StartTimeTest`, `FindEventCommandTest`, `ListEventCommandTest` and `DeleteEventTest` [\#231](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/231),
 
 * **Documentation**:
     * User Guide:
         * Added documentation for the `findEvents` command.
         * Removed documentation for the `tag` feature.
```

Comment:
`tag` is a field of person and not really a feature, so maybe this is better? Im also ok with not changing this
```suggestion
        * Removed documentation for the `tag` field in `addPerson` and `editPerson` features.
```